website/docs/r/log_analytics_data_export_rule.html.markdown

```diff
@@ -60,6 +60,14 @@ The following arguments are supported:
 
 * `enabled` - (Optional) Is this Log Analytics Data Export Rule enabled? Possible values include `true` or `false`. Defaults to `false`.
 
+* `metadata` - (Optional) A `metadata` block as defined below.
+
+---
+
+The `metadata` block exports the following attributes:
```

Comment:
```suggestion
The `metadata` block supports the following attributes:
```
Should this be `supports` instead of `exports`?

Reply:
Thanks. Updated